ingestion/src/metadata/ingestion/source/database/athena/metadata.py

```diff
@@ -57,6 +59,18 @@
 ATHENA_TAG = "ATHENA TAG"
 ATHENA_TAG_CLASSIFICATION = "ATHENA TAG CLASSIFICATION"
 
+ATHENA_INTERVAL_TYPE_MAP = {
+    **dict.fromkeys(["enum", "string", "VARCHAR"], PartitionIntervalTypes.COLUMN_VALUE),
+    **dict.fromkeys(
+        ["integer", "bigint", "INTEGER", "BIGINT"], PartitionIntervalTypes.INTEGER_RANGE
+    ),
+    **dict.fromkeys(
+        ["date", "timestamp", "DATE", "DATETIME", "TIMESTAMP"],
+        PartitionIntervalTypes.TIME_UNIT,
+    ),
+    "injected": PartitionIntervalTypes.INJECTED,
```

Comment:
are these injected partitions the `dynamic` ones from the issue?

Reply:
Yes that is correct